src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImport.java

```diff
@@ -169,6 +169,9 @@ public int getDefaultCloudIndexPrefetchBufferSize() {
         return 2;
     }
 
+    @Override
+    public String getProgressMeterRecordLabel() { return "batches"; }
```

Comment:
Well, this is really just a matter of semantics. "No batching" is just another way of saying "1 batch", for the purposes of this tool :)